README.md

```diff
@@ -67,38 +74,65 @@ The [track anatomy documentation](https://github.com/exercism/docs/blob/master/l
 ```bash
 ├── .gitignore
 ├── .github
-│ └── workflows
-│     └── tests.yml
+│   └── workflows
+│       └── tests.yml
 ├── LICENSE
 ├── README.md
 ├── bin
-│ └── fetch‐configlet
+│   └── fetch‐configlet
+├── concepts
+│   └── basics
+│   │   ├── about.md
+│   │   ├── introduction.md
+│   │   └── links.json
+│   └── ...
 ├── config.json
 ├── docs
-│ ├── ABOUT.md
-  ├── EXERCISE_README_INSERT.md
-│ ├── INSTALLATION.md
-│ ├── LEARNING.md
-│ ├── RESOURCES.md
-│ └── TESTS.md
+│   ├── ABOUT.md
+│   ├── EXERCISE_README_INSERT.md
+│   ├── INSTALLATION.md
+│   ├── LEARNING.md
+│   ├── RESOURCES.md
+│   └── TESTS.md
 └── exercises
-  └── accumulate
-  │ ├── package.yaml
-  │ ├── stack.yaml
-  │ ├── src
-  │ │ └── Accumuĺate.hs
-  │ ├── test
-  │ │ └── Tests.hs
-  │ └── .meta
-  │   ├── examples
-  │   │ └── success-standard
+  └── concept
+  │   ├── AnnalynsInfiltration
   │   │   ├── package.yaml
-  │   │   └── src
-  │   │     └── Accumulate.hs
-  │   └── hints.md
-  └── allergies
-  │ ├── ...
-  └── ...
+  │   │   ├── stack.yaml
+  │   │   ├── src
+  │   │   │   └── AnnalynsInfiltration.hs
+  │   │   ├── test
+  │   │   │   └── Tests.hs
+  │   │   ├── .docs
+  │   │   │   ├── instructions.md
+  │   │   │   ├── introduction.md
+  │   │   │   └── hints.md
+  │   │   └── .meta
+  │   │       ├── config.json
+  │   │       ├── design.md
+  │   │       └── exemplar
+  │   │           ├── package.yaml
+  │   │           └── src
+  │   │               └── AnnalynsInfiltration.hs
+  │   └── ...
+  └── practice
+    ├── accumulate
```

Comment:
vertical line extending under `practice` isn't aligned with first character (`p`) of `practice`.

I gather that the point of realigning this section is to consistently make the vertical line under a directory aligned with its first character, but the rule was broken here. Either revert the change and do no reformatting, or apply the new rule to `practice` as well.